keyboards/boston_meetup/config.h

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFB30
+#define PRODUCT_ID      0x26BE
+#define MANUFACTURER    ishtob
+#define PRODUCT         Boston Meetup Board
+#define DESCRIPTION     A limited-run coomunity meetup board
```

Comment:
"Community" is still misspelled.

```suggestion
#define DESCRIPTION     A limited-run community meetup board
```

Reply:
That's how they pronounce it in Boston